internal/module/conditions.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 const (
```

Comment:
Can we change the package name from `module` to `conditions`? I think `module` is a bit confusing here.